### DISCUSSION TITLE
GPS icon on bottom right blinking and toggles between red and black

### DISCUSSION THREAD

**Discussion by TheLastProject** [streetcomplete/StreetComplete#3304] [Q&A] [answered]
For me, the GPS icon in the bottom right in blinking. That makes sense to me: it doesn't have a location fix yet.

However, when I tap it the black GPS icon it becomes red, and on another tap becomes black again. All while still blinking. What does this mean?

**Comment by smichel17** [comment, marked as answer]
As you guessed, blinking means location services are enabled but your location has not yet been found.

The color indicates what will happen when the fix is found. If it is black: nothing (except that your position will be displayed). If it is red, the app will start following your position

On September 20, 2021 12:18:45 PM UTC, Sylvia van Os ***@***.***> wrote:
>For me, the GPS icon in the bottom right in blinking. That makes sense to me: it doesn't have a location fix yet.
>
>However, when I tap it the black GPS icon because red, and on another tap becomes black again. All while still blinking. What does this mean?
>

-- 
To respect your time, I try to write short, functional emails.
